homeassistant/components/squeezebox/media_player.py

```diff
@@ -156,7 +185,11 @@ async def _discovered_player(player):
             if entity and not entity.available:
```

Comment:
I think we should remove the available check and always update the player and send the signal with the player connected attribute as an argument.

Inside the entity we update available as appropriate based on the player connected state.

Reply:
I can make that change, I just want to make sure its not adding too much unnecessary overhead.

The server gets polled every 60 seconds for players. If we don't check `entity.available`, this signal will get sent for every 60 seconds for every connected player. Each `SqueezeBoxEntity` would then see the signal and check the `unique_id` to see if the signal applies to it.

If we do check `entity.available`, we won't be sending this signal very often.

Reply:
One signal per entity per every 60 seconds is fine. The entity shouldn't update state unless there's a change in availability so we won't spam the state machine.

Reply:
Great. And I realized that only unavailable players need to listen for this signal, which cuts it down even more.